JSDemos/Demos/DataGrid/BatchUpdateRequest/jQuery/index.js

```diff
@@ -0,0 +1,68 @@
+$(function () {
+    var URL = "https://js.devexpress.com/Demos/Mvc/api/DataGridWebApi";
+
+    $("#gridContainer").dxDataGrid({
+        dataSource: {
+            store: DevExpress.data.AspNet.createStore({
```

Comment:
You can assign store to dataSource directly without using json config { store: ... }